acados/ocp_nlp/ocp_nlp_dynamics_cont.c

```diff
@@ -246,6 +248,11 @@ void *ocp_nlp_dynamics_cont_memory_assign(void *config_, void *dims_, void *opts
 
     // adj
     assign_and_advance_blasfeo_dvec_mem(nu + nx + nx1, &memory->adj, &c_ptr);
+
+    // blasfeo_mem align
+    align_char_to(64, &c_ptr);
+    // hes
+    assign_and_advance_blasfeo_dmat_mem(nu+nx, nu+nx, &memory->hes, &c_ptr);
```

Comment:
Can't see the whole file here, but it's not so typical to assign a blasfeo_dmat in between dvecs. Can't this allocation be moved where the other dmats are allocated, or if this is the only one, right after the first alignment at 64 such that we don't need two of them?

Reply:
fixed it on my PR

Reply:
done

Reply:
wait, are you correcting as well? Should I wait for a few hours?

Reply:
ups. are we doing the same things? ;) Check my PR to your master

Reply:
I will stop now. Just started a few small things. You can merge my PR and continue if you like!

Reply:
(if it passes travis)